src/cbmc/cbmc_parse_options.cpp

```diff
@@ -541,6 +542,12 @@ bool cbmc_parse_optionst::set_properties()
       ::set_properties(goto_model, cmdline.get_values("property"));
   }
 
+  catch(const incorrect_goto_program_exceptiont &e)
```

Comment:
I don't like these try-catch blocks that do not perform any recovery action. In my opinion, all the try-catch blocks in cbmc_parse_options can be removed because the only action that is taken is to exit. The same can be achieved in a much cleaner way by catching those exceptions at top-level (parse_optionst) only.

Reply:
Yes, it's much neater catching them all in `parse_option_baset`.